src/simplewallet/simplewallet.cpp

```diff
@@ -2391,6 +2394,26 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
     payment_id_seen = true;
   }
 
+  uint64_t locked_blocks = 0;
+  if (transfer_type == TransferLocked)
+  {
+    try
+    {
+      locked_blocks = boost::lexical_cast<uint64_t>(local_args.back());
+    }
+    catch (const std::exception &e)
+    {
+      fail_msg_writer() << tr("bad locked_blocks parameter:") << " " << local_args.back();
+      return true;
+    }
+    if (locked_blocks > 1000000)
```

Comment:
It's a magic number, I don't think there's any that really makes sense. The original author's intent was to avoid "too large" values, but what's too large is pretty subjective.